src/components/Card/src/CardContent/mui-override.css

```diff
@@ -0,0 +1,13 @@
+.denhaag-card__content {
+    padding-block: 0 0;
```

Comment:
Why `0 0` instead just `0` here?

Reply:
Becausejust 0 won't override the specifity of the padding-bottom material-ui uses for the underlying cardcontent component